apple/Sources/Sargon/Extensions/Swiftified/Profile+Supporting+Types/AccountOrPersona+Swiftified.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  File.swift
+//  
+//
+//  Created by Alexander Cyon on 2024-04-27.
+//
+
+import Foundation
+import SargonUniFFI
+
+extension AddressOfAccountOrPersona: BaseEntityAddressProtocol {}
+
+extension AccountOrPersona {
+	public static func account(_ account: Account) -> Self {
+		.accountEntity(account)
+	}
+	
+	public static func persona(_ persona: Persona) -> Self {
+		.personaEntity(persona)
+	}
+}
+
+extension AccountOrPersona: EntityBaseProtocol {
```

Comment:
@micbakos-rdx I let the `AccountOrPersona` confirm to same base `EntityBaseProtocol` as `Account` and `Persona`.